src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -8695,6 +8695,71 @@ def test_aks_update_with_keda(self, resource_group, resource_group_location):
             self.is_empty(),
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_create_with_vpa(self, resource_group, resource_group_location):
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=77148984&view=results) to validate the change.

Reply:
Queued [new runs](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=77465557&view=results), please commit the recordings files (find from artifact) if test passed.